common/src/main/java/org/hyperskill/database/common/Request.java

```diff
@@ -0,0 +1,46 @@
+package org.hyperskill.database.common;
+
+import java.util.Objects;
+
+public class Request {
+    public enum TYPE {SET, GET, DELETE}
```

Comment:
For now it's okay, cause you use it only inside the `Request` class, but if you'd try to pass `TYPE` as a parameter somewhere else, you'll have to create an instance of 'Request' to access the enum. To avoid it make it `static`

Also, enum is just another class, and so follows the same naming convention (upper camel case, instead of all-caps). The values of enum are constants, and so they are written in caps, but not the enum itself